centos_ci.sh

```diff
@@ -224,6 +278,16 @@ else
     cd gopath/src/github.com/minishift/minishift
     make gen_adoc_tar IMAGE_UID=$(id -u)
     docs_tar_upload $PASS
+  elif [[ "$JOB_NAME" = "minishift-release" ]]; then
+    GIT_USER="minishift-bot"
+    prepare;
+    cd gopath/src/github.com/minishift/minishift
+    bump_to_release_version $GIT_USER;
+    make release
+    add_release_notes $GIT_USER;
+    # Generate docs tar and upload
+    make gen_adoc_tar IMAGE_UID=$(id -u)
+    docs_tar_upload $PASS
```

Comment:
let's send a notification to ' http://minibot.34e99f76.svc.dockerapp.io:9009/hubot/release' as well. Make it a JSON payload. You can use _curl_ to send it and you can use _jq_ to create the payload. 

I leave it up to you what you want to send. Maybe a single _message_ string or something more normalised - e.g. version, download_url, ...

We might want to add a "success" field as well. 

If you provide me the JSON payload format I can take care of the rest.